### DISCUSSION TITLE
Variants matching

### DISCUSSION THREAD

**Discussion by scienception** [PGScatalog/pgsc_calc#140] [Q&A] [answered]
Is the variant matching done using identifiers of SNPs or with physical position (chromosome number and base pair position)?

**Comment by smlmbrt** [comment, marked as answer]
Within the pipeline the variants are matched by chromosome, position, and the alleles (`effect_allele`, and when also provided the `other_allele`). However, when you're using the harmonised scoring files rsIDs may be used to get the chromosomal location for the variant (see https://www.pgscatalog.org/downloads/#dl_ftp_scoring_hm_pos for details).
